cf/read_write/netcdf/netcdfwrite.py

```diff
@@ -89,9 +135,12 @@ def _write_data(
         cfvar,
         ncvar,
         ncdimensions,
+        domain_axes=None,
         unset_values=(),
         compressed=False,
         attributes={},
+        construct_type=None,
+        warn_invalid=None,
```

Comment:
This doesn't seem to be used at all, isn't documented, and doesn't appear in the equivalent `cfdm` method, so I see nor reason for it to be there (was it left in from dev. work)?

```suggestion
```